src/module_2/EDA.md

```diff
@@ -0,0 +1,324 @@
+<h1 align="center">Exploratory Data Analysis</h1>
+
+Libraries
+
+
+```python
+import pandas as pd
+import pyarrow
+import matplotlib.pyplot as plt
+import seaborn as sns
+```
+
+Descargamos el DataSet y visualizamos las primeras filas que contiene:
+
+
+```python
+file_path = '/home/oscar/data/feature_frame.csv'
+df = pd.read_csv(file_path, low_memory=False)
+#print(df.head())
+```
+
+**Quick checks**
+
+
+```python
+print("Número de filas y columnas:", df.shape)
+```
+
+
+```python
+print(df.info())
+```
+
+Observamos que no hay valores nulos ni nan a priori en el Dataset para ninguna de las variables:
+
+
+```python
+print(df.isnull().sum())
+```
+
+
+```python
+print(df.describe())
+```
+
+Tras revisar los datos tenemos un conjunto con 27 características y 2880549 observaciones. Ninguna de las características presenta null-values. Evaluamos los diferentes datos y revisamos los datatypes:
+
+1. **Información general del pedido y usuario:**
+- 'order_id' : identificador único del pedido (int64)
+- 'user_id' : identificado único del usuario (int64)
+- 'created_at' : fecha y hora de creación del registro (object)
+- 'order_date' : fecha del pedido (object)
+
+El identificador único del pedido y del usuario debería ser tratado como una variable categórica y las diferentes fechas del pedido y de creación del registro deben de ser de tipo datetime:
```

Comment:
I'd strongly disagree.

The order ID is unique per order and therefore can't have any predictive power. User ID should never be used unless you want to build some kind of embeddings. Otherwise, the idea would be to enrich each instance with info from the user using only info prior to the instance (to avoid information leakage)